entrypoint/sonobuoyconformance/sonobuoyconformance_suite_test.go

```diff
@@ -0,0 +1,56 @@
+package sonobuoyconformance
```

Comment:
Can we rename the package to `conformance`, so suite will be `conformance_suite_test` and test file will be `sonobuoy_test.go`